tests/acceptance/features/bootstrap/FeatureContext.php

```diff
@@ -1206,13 +1220,13 @@ public function userHasSentHTTPMethodToUrlWithPassword(string $user, string $ver
 	 * @param string|null $user
 	 * @param string|null $verb
 	 * @param string|null $url
-	 * @param TableNode|null $body
+	 * @param TableNode|string|null $body
 	 * @param string|null $password
 	 *
-	 * @return void
+	 * @return ResponseInterface
 	 * @throws GuzzleException
 	 */
-	public function sendingToWithDirectUrl(?string $user, ?string $verb, ?string $url, ?TableNode $body, ?string $password = null): void {
+	public function sendingToWithDirectUrl(?string $user, ?string $verb, ?string $url, $body = null, ?string $password = null): ResponseInterface {
```

Comment:
missed this line
https://github.com/owncloud/ocis/blob/2e038038977fb857474591178cbaf80e2f13e879/tests/acceptance/features/bootstrap/FeatureContext.php#L1201